recipes/lucit-licensing-python/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "lucit-licensing-python" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 49eb84c50772d25e8723ac8d39c7cbae29ab90215d173d7b84fd1c2440fae9c7
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython
+    - pip
+  run:    
+    - python
+    - cython
+    - requests
+    - simplejson
+
+test:
+  imports:
+    - lucit_licensing_python
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python
+  summary: LUCIT Licensing Client Module
+  description: |
+    This module is used to verify LUCIT software licenses and also provides the developer with a command line interface to interact with the LUCIT Licensing API.
+
+    If you have already installed modules of LUCIT like the UNICORN Binance Suite, which use the lucit-licensing-python library, you have already installed the commandline tool lucitlicmgr automatically.
+  license: MIT
```

Comment:
Doesn't look like MIT: https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/blob/main/LICENSE#L1

Reply:
Thank you! "LSOSL" als license is not accepted. what can i do to solve this?

Reply:
What do you mean by not accepted?

Reply:
If i replace MIT with LSOSL in the recipe I get this hint:

> For recipes/lucit-licensing-python:
> 
>     License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

This just offers a limited list of open source licenses where mine is not inluded.

Reply:
ah, thanks, did not read correctly! I will try again. Yes i am sure the license fits, its my own project and i wrote the license myself.

Reply:
Ok, linter accepted it and other checks passed also. Thank you very much :D

Reply:
@xhochy do you mind taking a look at this license before we merge? I missed important details in the last one.

@oliver-zehentleitner consider using an OSI license for your projects. It makes the package adoption and distribution much easier.

Reply:
I think switching is not possible, OSI says that we are not allowed to charge a license fee. but we need to do that.

Reply:
Which important detail do you miss in the license i use?

Reply:
Well, if I knew I wouldn't be missing it :-)

I just want another set of eyes here b/c I did make a bad merge recently. Just want to avoid another mistake on my part.

Reply:
The LICENSE file only refers to a mail address regarding distribution. Overall, the LICENSE file does not state a license but only refers that you need an actual license to use it. I don't think we can ship this.

Reply:
@xhochy I reworked the license. Please review it! Thank you!

Reply:
I have no objections anymore but I also don't feel comfortable approving this. I have removed my ❌ though.

Reply:
I would be fine with distributing the license; my objection is that it does not allow usage without a license. As the software is used in the test, we would need a license that allows us to do so.

I am not a part of core though so just my 2 cents

Reply:
@xhochy Thank you!
@BastianZim That's right, this would actually be a license violation by definition. I will extend the license for this and get back to you.

Reply:
I added this to our license:
https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/blob/8bfd312fe7e59e4cb0457bb7ed6393ee29c650d4/LICENSE#L67

> Furthermore, this license explicitly permits Conda-Forge, an open-source community-driven platform for package 
> management and distribution operated by NumFOCUS, to test the software package following its creation to ensure 
> functionality, compatibility, and security standards are met. This permission is granted to support the community's 
> efforts in maintaining a robust ecosystem for software distribution and to guarantee the quality and performance of  
> the software for end users.
>

Reply:
That looks acceptable to me, but I would like to get a second pair of eyes on this.

@ocefpaf and @xhochy what do you think?

Reply:
@conda-forge-admin, please ping team